src/controllers/v1/_common/demand.ts

```diff
@@ -82,15 +86,56 @@ export class DemandController extends Controller {
     if (updated_since) {
       query.push(['updated_at', '>', parseDateParam(updated_since)])
     }
+    const selfAddress = (await this.addressResolver.determineSelfAddress()).address
+    const roles = this.env.ROLES
+    const demands: DemandRow[] = []
```

Comment:
I think it's possible remove pushing to an `demands` array, can be confusing keeping track what's in the array through the function